tests/compute/subresources/vmi.go

```diff
@@ -75,14 +76,14 @@ var _ = compute.SIGDescribe("VirtualMachineInstance subresource", func() {
 			)
 		})
 
-		It("[test_id:7476]Freeze without guest agent", func() {
+		It("[test_id:7476]Freeze without guest agent", decorators.Conformance, func() {
```

Comment:
This is too simple and should be unit test.

Reply:
Honestly I find it hard to convert it to a unit test.
Currently, there is no explicit logic that blocks the freeze for a vmi without guestagentconnected.
But it relies on the fact that the put request on the virt-handler side fails.
I can convert to unit test mocking the failing of virt-handler connection, but honestly it is not strictly related to guest agent connected. Should it works for you?